deploy/templates/default-azuredeploy-webjobs.json

```diff
@@ -410,7 +411,22 @@
           "keySource": "Microsoft.Storage"
         },
         "accessTier": "Hot"
-      }
+      },
+      "resources": [
+        {
+          "type": "Microsoft.Storage/storageAccounts/providers/roleAssignments",
+          "apiVersion": "2020-03-01-preview",
```

Comment:
Do we need to use the preview API?

Reply:
Can you try 2020-04-08?  That seems like the latest GA version.

---
In reply to: [564892943](https://github.com/microsoft/iomt-fhir/pull/87#discussion_r564892943) [](ancestors = 564892943)

Reply:
I tried 2020-04-08 but it said that version didn't support role assignments, so will instead go with the latest non-preview version which is 2018-07-01

Status Message: No registered resource provider found for location 'northcentralus' and API version '2020-04-08' for
type 'roleAssignments'. The supported api-versions are '2014-04-01-preview, 2014-07-01-preview, 2014-10-01-preview,
2015-05-01-preview, 2015-06-01, 2015-07-01, 2016-07-01, 2017-05-01, 2017-09-01, 2017-10-01-preview,
2018-01-01-preview, 2018-07-01, 2018-09-01-preview, 2018-12-01-preview, 2019-04-01-preview, 2020-03-01-preview,
2020-04-01-preview'.

Reply:
Thanks for checking Will.  We should probably follow up with the Azure Storage team.  We don't want to take a dependency on an API in preview if we can avoid it. Would you be able to create a work item for tracking?

---
In reply to: [567489805](https://github.com/microsoft/iomt-fhir/pull/87#discussion_r567489805) [](ancestors = 567489805)

Reply:
Nevermind I misread, it looks like 2018-07-01 works, thanks for the update.

---
In reply to: [569032564](https://github.com/microsoft/iomt-fhir/pull/87#discussion_r569032564) [](ancestors = 569032564,567489805)